datadog_checks_dev/datadog_checks/dev/tooling/utils.py

```diff
@@ -62,14 +62,50 @@ def string_to_toml_type(s):
     return s
 
 
+def complete_set_root(args):
+    """Set the root directory within the context of a cli completion operation."""
+
+    # if we have already set to something other than empty string break
+    if get_root() != '':
+        return
+
+    root = os.getenv('DDEV_ROOT', '')
+    if root and os.path.isdir(root):
+        set_root(root)
+    else:
```

Comment:
FWIW we could unnest this a bit using…

```python
if root and os.path.isdir(root):
    set_root(root)
    return

config = load_config()
...
```

Reply:
Thanks for review!  I wanted to get something out there to fix current issues, but will now try to refactor the base `ddev` command to utilize this function as well.

Reply:
I'd fix, then refactor in a separate PR if need be

Reply:
👍 - just posted fix, but will need refactor to get the env variable usable as part of the base command too.  Will post in separate PR.